tests/plugins/test_bump_release.py

```diff
@@ -129,9 +139,19 @@ def getNextRelease(self, build_info):
             def getBuild(self, build_info):
                 assert build_info['name'] == list(component.values())[0]
                 assert build_info['version'] == list(version.values())[0]
-                if build_info['release'] >= next_release['expected']:
-                    return None
-                return True
+
+                # Check 'major' part of the releases
+                if '.' in next_release['actual'] and \
```

Comment:
I don't think we want to perform splitting in the mock `getBuild`.
It should return build info solely based on `next_release['exists']`

I've altered the plugin code, as such:
```python
# next_release = self.get_patched_release(self.xmlrpc.getNextRelease(build_info))
next_release = self.xmlrpc.getNextRelease(build_info)  
```
and these tests still pass. They should fail.

I think the plugin code is right. It's just the testing that needs to be a little more robust.